descriptors/ModuleDescriptor-template.json

```diff
@@ -136,7 +136,7 @@
     },
     {
       "id": "inventory-batch",
-      "version": "0.2",
+      "version": "0.4",
```

Comment:
Are items included in the inventory-batch interface?

Reply:
> Are items included in the inventory-batch interface?

Sorry it's mistake. I'll fix it.